Modules/File/classes/class.ilObjFileGUI.php

```diff
@@ -375,7 +375,8 @@ protected function uploadFiles(): void
         if (self::UPLOAD_ORIGIN_DROPZONE === $origin) {
             $dropzone = new ilObjFileUploadDropzone($this->parent_id);
             $dropzone = $dropzone->getDropzone()->withRequest($this->request);
-            $files = $dropzone->getData()[self::PARAM_FILES] ?? null;;
+            $files = $dropzone->getData()[self::PARAM_FILES] ?? null;
+            ;
```

Comment:
:wink: Nitpick: extra line, caused by prior typo? Don’t think it marks an important spot in the code; let’s drop the extra semicolon (the new line 379), shall we?

Reply:
Is gone now!